docs/roadmaps/index.md

```diff
@@ -2,7 +2,11 @@
 
 # Roadmaps
 
-The [**napari global roadmap**](https://github.com/orgs/napari/projects/24/views/2?pane=info) is hosted on a Github Project board. This roadmap captures the current priorities of the napari core developer team. It is revisited every six months.
+The [**napari global roadmap**](active_roadmap.md) is a living document, and we revisit it every three months. It contains a summary of our vision, our mission and values, and our key focus areas
+for the project. It also details strategic deliverables and key priorities for each focus area within a horizon of 3-5 years.
+
+As a companion to the roadmap, we manage a [Github Project board](https://github.com/orgs/napari/projects/24/views/2?pane=info) that contains more technical details, issues and Pull-Requests
```

Comment:
do we want to just remove the bit about the project board? or do we want to commit to keeping it updated?

Reply:
Remove it. It can always be added later if needed.